### DISCUSSION TITLE
Gmail Issue with Captcha ( Winforms Cefsharp v105.3.390 )

### DISCUSSION THREAD

**Discussion by FredCousin** [cefsharp/CefSharp#4258] [General]
Greetings,

Since upgrading my application to cefsharp v105.3.390 i have the strange issue that Gmail always asks to fill in a captcha when trying to log in.
i don't have this issue on one of the previous versions of cefsharp.
I also tried different gmail accounts + Different places ( With different Ip addresses. ) 
When i log in on a older cefbrowser version i don't receive any captcha to fill in

When i log in on the new version of cefsharp i receive this captcha instant and it's every single log in.

Could you please check this and let me know?

By the way : many thanks for creating this + maintaining this!

Kind Regards,
Fred.

**Comment by amaitland** [comment]
[Google have been blocking Gmail logins](https://security.googleblog.com/2019/04/better-protection-against-man-in-middle.html) for a number of years for all embedded browsers. By default I'd expected a this browser might not be secure message. 

Are you using a workaround? Changed the user agent?
